### ISSUE TITLE
Sign in to localhost

### ISSUE THREAD

**Issue by ghost** [Odoo-mobile/framework#90]
Hi everyone, I'm new in both Odoo mobile and Android.
Before, I used "website" and "website_sale" module in Odoo 8 to make a website to receive orders from customers. And now I'm building a mobile application (an e-commerce application, not a management tool) with the same function. All is done in localhost:8069.
Can anyone suggest me how to connect with localhost? And am I possible to build this mobile application with Odoo mobile framework?

Thank you guys so much for reading. Hope anyone can give me a helping hand.

**Comment by dnia** [comment]
hi , what is your emulator? 
its possible

**Comment by iamphamduyhung** [comment]
I used the default emulator in Android Studio. I have successfully connected with the link 10.0.2.2:8069.

But I'm struggling building a mobile application like this one (https://www.odoo.com/nl_NL/page/e-commerce). All my functions were written in Python. Can I somehow call those methods using Java in the Odoo Mobile Framework or I have to rewrite them?

Thank you so much and sorry if I asked a stupid question.
